library/src/main/java/com/github/amlcurran/showcaseview/OnShowcaseEventListener.java

```diff
@@ -56,6 +58,12 @@ public void onShowcaseViewDidHide(ShowcaseView showcaseView) {
         public void onShowcaseViewShow(ShowcaseView showcaseView) {
 
         }
+
+        @Override
+        public void onShowcaseViewTouchBlocked(MotionEvent motionEvent) {
+
+        }
     };
 
+    void onShowcaseViewTouchBlocked(MotionEvent motionEvent);
```

Comment:
:-1: I'd move this before the `NONE` definition and add a javadoc description, as done for the other methods of this interface